member/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+from data_center.models import Course
+
+
+class Member(models.Model):
+    user = models.OneToOneField(User)
+
+    # Facebook field
+    uuid = models.CharField('User ID', max_length=100)
+    email = models.EmailField('Email')
+    access_token = models.TextField('access token')
```

Comment:
Yes, but it'll update each time as the user log in. (Refer to `member/views.py` Line.12 )
But we may not need this to get more information from facebook ? maybe I' ll remove this field.